components/MainLayout/components/shared/DesktopStyles.module.css

```
@@ -54,6 +66,64 @@
+  /* a label added later must overflow visibly in review, not silently wrap */
+  white-space: nowrap;
+
+  & > li:last-child {
+    margin-right: 0;
+  }
+
+  /* middot between adjacent links within a group */
+  & > li + li::before {
+    content: "\00B7";
+    display: inline-block;
+    color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)